analyzers/src/SonarAnalyzer.RuleDescriptorGenerator/SonarAnalyzer.RuleDescriptorGenerator.csproj

```
@@ -16,4 +16,13 @@
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="1.3.2" />
     <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic.Workspaces" Version="1.3.2" />
   </ItemGroup>
+
+  <Target Name="CopyBinaries" AfterTargets="Build">
+    <ItemGroup>
+      <BinariesToCopy Include="$(OutputPath)\*.dll" />
+      <BinariesToCopy Include="$(OutputPath)\*.json" />
```

Slightly more future proof:
```suggestion
      <BinariesToCopy Include="$(OutputPath)\**\*.dll" />
      <BinariesToCopy Include="$(OutputPath)\**\*.json" />
```

---

This tiny project should not get any bigger any time soon (I've deleted most of it last year)